resources/nl/cemu.po

```diff
@@ -1,386 +1,501 @@
+#: CemuUpdateWindow.cpp
 msgid "Exit"
 msgstr "Afsluiten"
 
-#: CemuUpdateWindow.cpp:507
+#: CemuUpdateWindow.cpp
 msgid "No update available!"
 msgstr "Geen update beschikbaar!"
 
-#: CemuUpdateWindow.cpp:526
+#: CemuUpdateWindow.cpp
 msgid "Update available!"
 msgstr "Update beschikbaar!"
 
-#: CemuUpdateWindow.cpp:531
+#: CemuUpdateWindow.cpp
 msgid "Extracting update..."
 msgstr "Uitpakken van update..."
 
-#: CemuUpdateWindow.cpp:536
+#: CemuUpdateWindow.cpp
 msgid "Couldn't download the update!"
 msgstr "Downloaden van update is niet gelukt!"
 
-#: CemuUpdateWindow.cpp:539
+#: CemuUpdateWindow.cpp
 msgid "Applying update..."
 msgstr "Bijwerken van bestanden"
 
-#: CemuUpdateWindow.cpp:547
+#: CemuUpdateWindow.cpp
 msgid "Extracting failed!"
 msgstr "Gefaald met uitpakken!"
 
-#: CemuUpdateWindow.cpp:553 MainWindow.cpp:438
+#: CemuUpdateWindow.cpp MainWindow.cpp
 msgid "Success"
 msgstr "Succes"
 
-#: CemuUpdateWindow.cpp:554
+#: CemuUpdateWindow.cpp
 msgid "Restart"
 msgstr "Herstarten"
 
-#: CemuUpdateWindow.cpp:578
+#: CemuUpdateWindow.cpp
 msgid "Downloading update..."
 msgstr "Update aan het downloaden..."
 
-#: DebuggerWindow2.cpp:170
+#: DebuggerWindow2.cpp
 msgid "GoTo (CTRL + G)"
 msgstr "Ga Naar (CTRL + G)"
 
-#: DebuggerWindow2.cpp:173
+#: DebuggerWindow2.cpp
 msgid "Toggle Breakpoint (F9)"
 msgstr "Wissel breekpunt (F9)"
 
-#: DebuggerWindow2.cpp:178 DebuggerWindow2.cpp:425
+#: DebuggerWindow2.cpp
 msgid "Break (F5)"
 msgstr "Stop (F5)"
 
-#: DebuggerWindow2.cpp:180
+#: DebuggerWindow2.cpp
 msgid "Step Into (F11)"
 msgstr "Stap in (F11)"
 
-#: DebuggerWindow2.cpp:181
+#: DebuggerWindow2.cpp
 msgid "Step Over (F10)"
 msgstr "Stap over (F10)"
 
-#: DebuggerWindow2.cpp:414
+#: DebuggerWindow2.cpp
 msgid "Run (F5)"
 msgstr "Uitvoeren (F5)"
 
-#: DebuggerWindow2.cpp:543
-msgid "E&xit"
-msgstr "Afsluiten"
+#: DebuggerWindow2.cpp MainWindow.cpp
+msgid "&Exit"
+msgstr "&Verlaten"
 
-#: DebuggerWindow2.cpp:546 MainWindow.cpp:2287
+#: DebuggerWindow2.cpp MainWindow.cpp
 msgid "&File"
 msgstr "&Bestand"
 
-#: DebuggerWindow2.cpp:550
+#: DebuggerWindow2.cpp
 msgid "&Pin to main window"
 msgstr "&Vastmaken aan hoofd venster"
 
-#: DebuggerWindow2.cpp:552 MainWindow.cpp:2336
+#: DebuggerWindow2.cpp MainWindow.cpp
 msgid "&Options"
 msgstr "&Opties"
 
-#: DebuggerWindow2.cpp:556
+#: DebuggerWindow2.cpp
 msgid "&Registers"
 msgstr "&Registers"
 
-#: DebuggerWindow2.cpp:557
+#: DebuggerWindow2.cpp
 msgid "&Memory Dump"
 msgstr "&Geheugen Dump"
 
-#: DebuggerWindow2.cpp:558
+#: DebuggerWindow2.cpp
 msgid "&Breakpoints"
 msgstr "&Breekpunten"
 
-#: DebuggerWindow2.cpp:559
+#: DebuggerWindow2.cpp
 msgid "Module&list"
 msgstr "Module&lijst"
 
-#: DebuggerWindow2.cpp:561
+#: DebuggerWindow2.cpp
 msgid "&Window"
 msgstr "&Venster"
 
-#: DisasmCtrl.cpp:501
+#: debugPPCThreadsWindow.cpp
+msgid "PPC threads"
+msgstr "PPC threads"
+
+#: debugPPCThreadsWindow.cpp
+msgid "Entry"
+msgstr "Item"
+
+#: debugPPCThreadsWindow.cpp
+msgid "Stack"
+msgstr "Stapel"
```

Comment:
It's better to leave them untranslated since they are technical strings related to Cemu internals.